addons/stock/security/ir.model.access.csv

```diff
@@ -81,3 +81,5 @@ access_stock_track_confirmation,access.stock.track.confirmation,model_stock_trac
 access_stock_track_line,access.stock.track.line,model_stock_track_line,stock.group_stock_user,1,1,1,0
 access_stock_package_destination,access.stock.package.destination,model_stock_package_destination,stock.group_stock_user,1,1,1,0
 access_stock_orderpoint_snooze,access_stock_orderpoint_snooze,model_stock_orderpoint_snooze,stock.group_stock_user,1,1,1,1
+access_stock_package_type_user,access_stock_package_type_user,model_stock_package_type,base.group_user,1,0,0,0
```

Comment:
Why not the stock user instead ?

Reply:
@Whenrow because product.packaging uses group_user. Do you think it's better to change it to stock_user?

Reply:
I think it's better to restrict to stock user. Packaging was defined in product module so you can't set stock user in that case